index.bs

```diff
@@ -1549,7 +1550,7 @@ Methods</h4>
 
 			1. Attempt to <a>release system resources</a>.
 
-			2. Set the <a>rendering thread state</a> to <code>suspended</code>.
+			2. Set the {{[[rendering thread state]]}} to <code>suspended</code>.
```

Comment:
Note to self: We should probably change `<code>suspended</code>` to use bikeshed markup to refer to `{{AudioContextState/suspend|suspended||`.  Assuming I got the syntax right.